site/docs/samples/notebook-read.md

```diff
@@ -61,20 +61,99 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
+          secret_key: "${SECRET_KEY}"
+        EOF
+        ```
+
+        Next, register the data asset itself in the data catalog.
+        We use port-forwarding to send asset creation requests to the OpenMetadata connector.
+        ```bash
+        kubectl port-forward svc/openmetadata-connector -n fybrik-system 8081:8080 &
+        ```
+        ```bash
+        cat << EOF | curl -X POST localhost:8081/createAsset -d @-
+        {
+          "destinationCatalogID": "openmetadata",
+          "destinationAssetID": "paysim-csv",
+          "credentials": "/v1/kubernetes-secrets/paysim-csv?namespace=fybrik-notebook-sample",
+          "details": {
+            "dataFormat": "csv",
+            "connection": {
+              "name": "s3",
+              "s3": {
+                "endpoint": "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566",
+                "bucket": "demo",
+                "object_key": "PS_20174392719_1491204439457_log.csv"
+              }
+            }
+          },
+          "resourceMetadata": {
+            "name": "Synthetic Financial Datasets For Fraud Detection",
+            "geography": "theshire ",
+            "tags": {
+              "Purpose.finance": "true"
+            },
+            "columns": [
+              {
+                "name": "nameOrig",
+                "tags": {
+                  "PII.Sensitive": "true"
+                }
+              },
+              {
+                "name": "oldbalanceOrg",
+                "tags": {
+                  "PII.Sensitive": "true"
+                }
+              },
+              {
+                "name": "newbalanceOrig",
+                "tags": {
+                  "PII.Sensitive": "true"
+                }
+              }
+            ]
+          }
+        }
+        EOF
+        ```
 
-Datasets can be registered either directly, through the OpenMetadata UI, or indirectly, through the data-catalog connector:
+        The response from the OpenMetadata connector should look like this:
+        ```bash
+        {"assetID":"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""}
+        ```
+        Store the asset ID in a `CATALOGED_ASSET` variable:
+        ```bash
+        CATALOGED_ASSET="openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""
+        ```
 
-=== "Register an asset through the OpenMetadata UI"
-    To register an asset directly through the OpenMetadata UI, follow the instructions [here](../../tasks/omd-discover-s3-asset/). These instructions also explain how to determine the asset ID.
-
-    Store the asset ID in a `CATALOGED_ASSET` variable. For instance:
-    ```bash
-    CATALOGED_ASSET="openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""
-    ```
-
-=== "Registering Dataset via Connector"
-    We now explain how to register a dataset using the OpenMetadata connector.
+=== "With Katalog"
+    We now explain how to register a dataset using the Katalog connector.
```

Comment:
We don't have Katalog connector, so we?
maybe change the sentence as it was in 1.2.1 
`We now explain how to register a dataset in the Katalog data catalog.`
It is not perfect but maybe less misleading.